packages/story-editor/src/components/canvas/layout.js

```diff
@@ -76,7 +76,7 @@ const Layer = styled.section`
     f = forward navigation
     p = canvas page
     m = page action menu
-    c = thumbnail carousel
+    w = workspace footer
```

Comment:
A better name than _Workspace footer_ is very welcome. I would prefer a single word. I considered things like _Utilities_ or _Navigation_, but neither really works for me. _Toolbox_ perhaps?

Reply:
_Toolbox_ is growing on me now that I think more about it...

Reply:
Just _footer_ perhaps?

Reply:
We're using just "header" for the header so just "footer" would work for consistency.

Reply:
Renamed to _Footer_ in 43b812a.